apolline-flutter/lib/models/indoor_user_actions.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:easy_localization/easy_localization.dart';
+
+/// Enum that represents indoors user actions
+/// TODO complete with the Jérôme's list (see meeting of 21/01/2022)
```

Comment:
Has this been done?

Reply:
No, I didn't get the list.

Reply:
It's not too late to send Jérôme an email :)